src/main/config/config.c

```diff
@@ -484,9 +484,6 @@ if (systemConfig()->configurationState == CONFIGURATION_STATE_UNCONFIGURED) {
 #ifdef USE_DASHBOARD
     featureEnableImmediate(FEATURE_DASHBOARD);
 #endif
-#ifdef USE_GPS
-    featureEnableImmediate(FEATURE_GPS);
-#endif
```

Comment:
This code has been present in 4.5 for 8 months without issues.  Are you sure it should be removed?

Reply:
`FEATURE_GPS` will be enabled by configuring the port or importing diff / preset and should optimize code in the case no GPS was configured.

Reply:
OK that makes sense